Foundation/NSURLSession/NSURLSessionTask.swift

```diff
@@ -838,9 +841,14 @@ extension URLSessionTask {
             var data = Data(capacity: bodyData!.length)
             data.append(Data(bytes: bodyData!.bytes, count: bodyData!.length)) 
 
+            #if !os(Android)
             s.delegateQueue.addOperation {
                 completion(data, response, nil)
             }
+            #else
+            // HACK: Operation Queues not working on Android
+            completion(data, response, nil)
+            #endif
```

Comment:
What's the plan to make sure dispatch and operation queues are actually working?